src/split/split.cpp

```diff
@@ -68,34 +68,35 @@ models parse_multimodel_pdbqt(const std::string& input) {
 	std::string str;
 	bool parsing_model = false;
 	bool parsing_ligand = true;
+
 	while(std::getline(in, str)) {
 		++count;
 		if(starts_with(str, "MODEL")) {
 			if(parsing_model == true || parsing_ligand == false) 
-				throw parse_error(p, count, "Misplaced MODEL tag");
+				throw pdbqt_parse_error("Misplaced MODEL tag at line " + std::to_string(count) + ".\n");
```

Comment:
Interesting ... now there's a newline at the end - while in parse_pdbqt.cpp there isn't ... Please make this consistent.